examples/browser/package.json

```diff
@@ -36,7 +36,7 @@
   },
   "scripts": {
     "prepare": "theia build --mode development",
-    "start": "TRACE_SERVER_PATH=../../trace-compass-server/tracecompass-server theia start --plugins=local-dir:../plugins",
+    "start": "cross-env TRACE_SERVER_PATH=../../trace-compass-server/tracecompass-server OVERVIEW_DEFAULT=Histogram theia start --plugins=local-dir:../plugins",
```

Comment:
We don't need to use an environment variable for OVERVIEW_DEFAULT. A preference is sufficient.

Btw, the environment variable for the trace server path exists to be able to package the trace server application when building the electron app. There is no need for an environment variable for the default overview.